aqua/QtPass/Portfile

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               github 1.0
+PortGroup               qt5 1.0
+PortGroup               qmake5 1.0
+
+github.setup            IJHack QtPass 1.4.0 v
+revision                0
```

Comment:
please add "github.tarball_from" line and set it to "releases (preferred if upstream provides them), otherwise "archive". That will require you to update the "checksums"